test/rules/arrow-parens/ban-single-arg-parens/test.ts.fix

```diff
@@ -38,3 +40,4 @@ let foo = bar => bar;
 
 let foo = async bar => bar;
 
+arr.map(a => b);
```

Comment:
this will probably fail for `foo(async (a) => b)` by fixing it to `foo(asyca => b)`